packages/flame_audio/example/lib/main.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: public_member_api_docs
```

Comment:
can't we override the rule on the example project analysis options? I think it's bad to have to ignore a rule on every example file. the rule just doesn't apply to the project